fields/types/s3files/S3FilesType.js

```diff
@@ -0,0 +1,572 @@
+/*!
+ * Module dependencies.
+ */
+
+var _ = require('underscore'),
+	moment = require('moment'),
+	keystone = require('../../../'),
+	util = require('util'),
+	knox = require('knox'),
+	// s3 = require('s3'),
```

Comment:
The "s3" require? Guess not. [Knox](https://github.com/Automattic/knox) does the S3 work.